### PR TITLE
Stub a linux project which compiles

### PR DESCRIPTION
**Pull request by isJuhn**
The final of the KAMICore trilogy. Fix namespaces, make core compile on linux, create linux console application.

### CODE REVIEW

File: KAMI.Core/KAMI.Core.csproj [hunk 1]
```diff
@@ -12,13 +20,10 @@
...
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsWindows)'=='true'">
     <Reference Include="WindowsInput">
       <HintPath>WindowsInput.dll</HintPath>
     </Reference>
-  </ItemGroup>
-
-  <ItemGroup>
     <None Update="pine_c.dll">
```
Comment (dio-gh) [inline KAMI.Core/KAMI.Core.csproj:27]: Should this not be adjusted for the `.so` option also?

**Review by dio-gh — APPROVED** [review]
Looks alright at a glance to the extent I can grasp it.

File: KAMI.Core/KAMI.Core.csproj (hunk 1 above)
Comment (dio-gh) [reply KAMI.Core/KAMI.Core.csproj:27]: That is a very deceptively selected preview GitHub.

I mean the `<None Update="pine_c.dll">` line.

Comment (isJuhn) [reply KAMI.Core/KAMI.Core.csproj:27]: this .dll is moved into conditional itemgroup for windows. .so is added to a new conditional itemgroup for linux a few lines below

Comment (dio-gh) [reply KAMI.Core/KAMI.Core.csproj:27]: Oh right, I see it now. Coolio then 👌

**Review by dio-gh — APPROVED** [review]

File: KAMI.Core/KAMI.Core.csproj (hunk 1 above)
Comment (dio-gh) [reply KAMI.Core/KAMI.Core.csproj:27]: Oh yea I can see it now, coolio.

**Pull request merged**